peer_mentoring/peer_mentoring/settings.py

```diff
@@ -163,3 +163,12 @@
 REDIS_HOST = "localhost"
 REDIS_PORT = 6379
 REDIS_DB = 0
+
+# SENDGRID API
+
+SENDGRID_API_KEY = os.getenv("SENDGRID_API_KEY")
+EMAIL_HOST = "smtp.sendgrid.net"
+EMAIL_HOST_USER = "apikey"
+EMAIL_HOST_PASSWORD = os.getenv(SENDGRID_API_KEY)
```

Comment:
I only have this one

Reply:
As in you only use this setup? Set to this, but not sending myself an email.

Reply:
I have `EMAIL_HOST_PASSWORD = os.getenv(SENDGRID_API_KEY)`, I don't have `SENDGRID_API_KEY = os.getenv("SENDGRID_API_KEY")`, you probably only need `EMAIL_HOST_PASSWORD`

Reply:
ok seeing your other comment you already deleted it, good